From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Sat, 17 Feb 2018 18:41:56 -0500
Subject: look for modules in python3

When building, we want to make sure we use python3 where possible, not
python 2.7. This is what the executable is named in debian.
---
 platform.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/platform.mk b/platform.mk
index 170269c..63a305d 100644
--- a/platform.mk
+++ b/platform.mk
@@ -200,7 +200,7 @@ endef
 
 # Find Python package
 define find_pythonpkg
-	HAS_$(1) := $(shell python -c "import $(1)" > /dev/null 2>&1 && echo yes || echo no)
+	HAS_$(1) := $(shell python3 -c "import $(1)" > /dev/null 2>&1 && echo yes || echo no)
 endef
 
 define find_soname
